### PR TITLE
Description Book Update

### PR DESCRIPTION
**Pull request by John-Dang**
## About The Pull Request
I changed descriptions and titles and removed the PA skillbook from the private diary, which is now called a Burned Book, etc... general book flavor changes! I also changed the remarks that pop up as you read the book to be actually helpful tips, mostly.

## Why It's Good For The Game
It removes power armor from being a thing any average waster can wear from roundstart as a skill and makes the books better!

## Changelog
:cl:
add: Added new book names and descriptions and remarks.
tweak: No more spawning with power armor training as a choosable trait!
/:cl:

### CODE REVIEW

**Comment by Pebblesthe2nd** [comment]
Overall I like the titles, just a couple things.

I'd suggest just leaving the Forgemaster one as is, the Phoenix title is to be honest not very good. Working on a way to make it obsolete anyways so not a big deal, but still, better just leave it alone.

I personally like the Little League book giving Big Leagues trait as a joke more than Grognak, but with the other books in line with Fallout ones I can understand changing it, even if I think its a slight downgrade in funny. Not like it was a master joke to begin with so hardly matters. Just a taste opinion.

Private diary becoming burned book I don't quite get the point of, unless there is some reason for it I'd prefer to not change names of stuff thats not misleading or shoddily written.

File: code/game/objects/items/granters.dm [hunk 1]
```diff
@@ -789,114 +789,86 @@
 	crafting_recipe_types = list(/datum/crafting_recipe/scoutcarbine)
 
 /obj/item/book/granter/trait/forgemaster
-	name = "The secrets of the Legion Forgemasters"
-	desc = "All you need to bend the steel right."
+	name = "Ashes of Phoenix: Imperial Weaponsmithing"
```
Comment (Pebblesthe2nd) [inline code/game/objects/items/granters.dm:792]:
```suggestion
	name = "The secrets of the Legion Forgemasters"
```

Please leave the title as before, rest is fine.

Comment (John-Dang) [reply code/game/objects/items/granters.dm:792]: The old title also isn't good, a new title is needed I believe. Have any suggestions on what it should be?

**Comment by Pebblesthe2nd** [comment]
One thing about deleting Teachings of Mars and Machine spirits books, they've been used to give skills to loadouts, never meant to be loot stuff. They are currently kind of obsolete, and not exactly hard to recreate if needed, but it would be better to just comment them out and place them in the bottom of the file until the current system with loadout books can be changed, to save effort in case a loadout change makes the books useful. Books are used since the grant trait code used dont allow direct trait adding to loadouts, just to roles, so they have a utility at times, clunky as they are.

File: code/game/objects/items/granters.dm (hunk 1 above)
Comment (Pebblesthe2nd) [reply code/game/objects/items/granters.dm:792]: Something simple is enough, its a type of book thats just more or less placeholder due to limitations in our current loadout code. "Forgemaster manual" maybe. Not worth too much effort or flourish, simple descriptive is fine for that style of loadout book, hopefully we can make them truly obsolete soon.

**Comment by John-Dang** [comment]
> One thing about deleting Teachings of Mars and Machine spirits books, they've been used to give skills to loadouts, never meant to be loot stuff. They are currently kind of obsolete, and not exactly hard to recreate if needed, but it would be better to just comment them out and place them in the bottom of the file until the current system with loadout books can be changed, to save effort in case a loadout change makes the books useful. Books are used since the grant trait code used dont allow direct trait adding to loadouts, just to roles, so they have a utility at times, clunky as they are.

If you'll suggest them back in as commented out, I'll accept that back into the code.

**Comment by John-Dang** [comment]
Seems ready for merge.

**Pull request merged**